.eslintrc.yml

```diff
@@ -0,0 +1,28 @@
+env:
+  es6: true
+  node: true
+
+extends:
+  - eslint:recommended
+
+root: true
```

Comment:
Nothing _too_ exciting in this file. I added an ".yml" extension to the .eslintrc file, per latest ESLint recommendations (plus why not).

Also changed all the "0", "1", "2" codes to the newer verbose syntax of "off", "warn", "error", respectively. Makes for better reading.